tsconfig.eslint.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "types": ["@types/node"],
+    "noEmit": true,
+    "allowJs": true
```

Comment:
Do we have javascript files in the source tree?

Reply:
this is just for the `eslint.config.js`